metrics/metrics-core/src/main/java/io/servicecomb/metrics/core/EmbeddedMetricsName.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2017 Huawei Technologies Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.servicecomb.metrics.core;
+
+public class EmbeddedMetricsName {
+
+  private static final String QUEUE_HEADER_FORMAT = "servicecomb.%s.queue";
+
+  public static final String QUEUE_COUNT_IN_QUEUE = QUEUE_HEADER_FORMAT + ".waitInQueue.count";
```

Comment:
why 2 QUEUE in the name? maybe `COUNT_IN_QUEUE_TEMPLATE`?

Reply:
Fixed